src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -373,36 +377,29 @@ private Vector limitDirectionSlewRate(Vector spdVec) {
     double angleDif =
         SwerveUtils.AngleDifference(inputTranslationDir, m_currentTranslationDirRadians);
 
-    m_prevSlewRateTime = currentTime;
-
     if (angleDif < DriveConfig.MIN_ANGLE_SLEW_RATE) {
       m_currentTranslationDirRadians =
           SwerveUtils.StepTowardsCircular(
               m_currentTranslationDirRadians, inputTranslationDir, directionSlewRate * elapsedTime);
       m_currentTranslationMag = m_magLimiter.calculate(inputTranslationMag);
-      return (new Vector(m_currentTranslationMag, 0)).rot(m_currentTranslationDirRadians);
-    }
-
-    if (angleDif > DriveConfig.MAX_ANGLE_SLEW_RATE) {
-      if (SwerveUtils.approxEqual(
```

Comment:
Why was this removed?

Reply:
approxEqual was switched to a direct comparison to avoid incorrect floating point approximations

Reply:
Approx equal does the comparison you are doing right now.

```java
class SwerveUtils {
	public boolean approxEqual(double x, double y) {
		return Math.abs(x - y, 1e-6);
	}
}
```

Reply:
Technically yes, but switching it to the default rev code solution actually gets the slew rate limiter to a nonzero magnitude. We discussed this in depth today, so even if the current code is still a bit of a workaround, it works while still implementing the slew rate limiter, so we decided it's best to keep it for now given time constraints.

Reply:
Okay then so long as it was discussed.